### DISCUSSION TITLE
Usability Verbesserungen

### DISCUSSION THREAD

**Discussion by hokascha** [OpenLigaDB/OpenLigaDB-Samples#90] [Ideas]
Ein paar Ideen zur Usablity bei der Verwaltung von Ligen:

1. Beim Hinzufügen neuer Spiele zu einem Spieltag:
1.1. Die Dropdown-Felder für die Teams sind unpraktisch weil zu kurz, die Suche per Texteingabe ist zu langsam, per Maus muss man immer hin und her scrollen und ewig das passende Team suchen. Vorschlag: Optional kein Dropdown, sondern eine **Liste aller Teams**. Die wird dann zwar lang, aber wir haben ja große Monitore ;-)
1.2. Die Liste der Teams bzw. die Drop-Downs könnten sich automatisch **kürzen**. Wenn Teams bereits am gleichen Spieltag spielen, könnten Sie für nachfolgende Spiele aus der Auswahl entfernt werden. Weniger suchen.
2. Beim Verwalten der Spieltage werden neue Spieltage immer hinten angehangen. Das ist ok, aber es wäre praktisch, sie per **Drag&Drop in die richtige Reihenfolge** bringen zu können. Aktuell muss immer die Order-ID des Spieltags manuell angepasst werden (und auch die nachfolgender Spieltage).
3. Auf kleinen Bildschirmen überlagert die Werbung den "Spielansetzungen anpassen"-Dialog.
4. Der "Spielansetzungen anpassen"-Dialog darf gerne breiter sein, sodass Datum/Uhrzeit vollständig zu sehen sind.

**Comment by msiggi** [comment]
Hier habe ich etwas getan:

> Beim Hinzufügen neuer Spiele zu einem Spieltag:
> 1.1. Die Dropdown-Felder für die Teams sind unpraktisch weil zu kurz, die Suche per Texteingabe ist zu langsam, per Maus muss man immer hin und her scrollen und ewig das passende Team suchen. Vorschlag: Optional kein Dropdown, sondern eine Liste aller Teams. Die wird dann zwar lang, aber wir haben ja große Monitore ;-)
> 1.2. Die Liste der Teams bzw. die Drop-Downs könnten sich automatisch kürzen. Wenn Teams bereits am gleichen Spieltag spielen, könnten Sie für nachfolgende Spiele aus der Auswahl entfernt werden. Weniger suchen.

Ich bitte um Feedback!
